### DISCUSSION TITLE
Deployment queued

### DISCUSSION THREAD

**Discussion by kaustabhws** [vercel/community#3791] [Help]
### Summary

I deployed my vercel app 1 day ago. It says deployment queued. I'm on hobby plan, and I don't have any other app on deployment process. Then I deleted the project and again tried to deploy. It's still in queued process.

### Steps to Reproduce

Help me with the deployment queued bug. I can't deploy any app

**Comment by amyegan** [comment]
Hey @kaustabhws. If you're still having this problem, can you share a link to the deployment that is stuck? That will let me look it up to get more information about why it isn't deploying normally.

**Reply by kaustabhws** [reply]
After 2 days, it's deployed now. After too many reattempts to deploy

[a comment by an automated account is not shown]
